source/05-pages/homepage.stories.jsx

```diff
@@ -55,7 +59,9 @@ const homepageContent = twigTemplate({
   homepage_grid_title: 'You Don’t Vote For Kings',
 });
 
-const Homepage = () => <PageWrapper>{parse(homepageContent)}</PageWrapper>;
-Homepage.args = {};
+const Homepage = args => <PageWrapper>{parse(homepageContent(args))}</PageWrapper>;
```

Comment:
Codacy has a fix for the issue: [Function component is not a function expression](https://app.codacy.com/gh/forumone/gesso/pullRequest?prid=8642047)
```suggestion
var Homepage = function(args) {
```

Reply:
We use that convention pretty often. I’ll tackle all of those cases next week.

Reply:
I don't think Codacy's suggestion is right here. If we want to switch to a function declaration, it should be:
```
function Homepage(args) {
...
}
```
We definitely don't want to use `var` in place of `const`. Not sure why Codacy's suggesting that.